jest.config.js

```diff
@@ -1,7 +1,9 @@
 module.exports = {
+	preset: 'ts-jest',
```

Comment:
Now that `ts-jest` is being used, the `.test.ts` files don't need to be compiled to JavaScript. Consider adding them to the `"exclude": []` list in `tsconfig.json`.

Reply:
That makes sense! Thx for your review!